contracts/DetherCore.sol

```diff
@@ -121,6 +143,10 @@ contract DetherCore is DetherSetup, ERC223ReceivingContract, SafeMath {
     isStarted = true;
   }
 
+  function setPriceOracle (address _priceOracle) external onlyCEO {
```

Comment:
Should be onlyCFO , create this new role in detherAccess

Reply:
what about line 286, should it also be CFO?

```
function setSellDailyLimit(uint _tier, bytes2 _countryId, uint256 _limitUsd) public onlyCSO {
  limitTier[_tier][_countryId] = _limitUsd;
}
```

Reply:
Yes all thing related to the limit should be CFO